gsoc.yml

```diff
@@ -1,3 +1,31 @@
+gsoc2018:
+  MananAgarwal:
+    rss_feed: "https://medium.com/feed/@manan_agarwal"
+    project: 'Astropy'
+  sushobhana:
+    rss_feed:  "https://medium.com/feed/@sushobhanapatra"
+    project: 'Astropy'
+  rkiman:
+    rss_feed: "https://rociokiman.blogspot.com/feeds/posts/default?alt=rss" # set up tag? FIXME?
+    project: 'Astropy'
+  ritiek:
+    rss_feed: "https://ritiek.github.io/feed.xml"  # set up tag? FIXME?
```

Comment:
@ritiek Would be possible to get a feed for only the posts you would do related with GSoC?